sunpycube/cube/datacube.py

```diff
@@ -75,28 +77,42 @@ def __init__(self, data, wcs, errors=None, **kwargs):
         # Also it's called axes_wcs because wcs belongs to astropy.nddata and
         # that messes up slicing.
 
-    def plot_wavelength_slice(self, offset, axes=None,
-                              style='imshow', **kwargs):
+    def plot_wavelength_slice(self, offset, axes=None, unit_x_axis=None,
```

Comment:
We should not expose the axis units in this API. The animator class is a special case because the axes object is never exposed to the user. Here, like in sunpy map, the correct thing for the user to donis use the WCSAxes API.